### PR TITLE
add indexPrefix customization for ElasticSearch log drain

### LINKED ISSUE #695: [logs] support indexPrefix optional field for ElasticSearch drains
The ElasticSearch drain has a default publishing index `logstash-<date>`. API has been updated to support passing `indexPrefix` to the related drain creation.

I'm willing to open a PR.

### PR DESCRIPTION
**Pull request by KannarFr**
Closes #695.

I'm sure you have a proper way to optionally print the index in the drain list ;).

### CODE REVIEW

**Comment by aurrelhebert** [comment]
@KannarFr how do handle QA on this PR?

We block request merge on fixup commit (https://github.com/CleverCloud/clever-tools/pull/684),  and it looks like it didn't generate the snapshots. Could you push a new commit instead? Thanks

**Comment by KannarFr** [comment]
Create an ES drain with an indexPrefix, I already did it and it's ok. 

As the commit I reworked is just a typo the existing preview is ok.

**Comment by KannarFr** [comment]
Nvm, misread your message, done.

File: src/commands/drain.js [hunk 1]
```diff
@@ -25,16 +25,22 @@ async function list (params) {
...
 
   drains.forEach((drain) => {
-    const { id, state, target: { url, drainType } } = drain;
-    Logger.println(`${id} -> ${state} for ${url} as ${drainType}`);
+    const { id, state, target } = drain;
+    const { url, drainType, indexPrefix } = target;
+
+    let drainView = `${id} -> ${state} for ${url} as ${drainType}`;
+    if (indexPrefix != null) {
+      drainView += `, index: \`${indexPrefix}-<YYYY-MM-DD>\``;
```
Comment (miton18) [inline src/commands/drain.js:33]: is it an index name, or ... an index name template ?

**Comment by pdesoyres-cc** [comment]
Nice improvement.

I'm wondering if you thought about adding some documentation in `docs/log-drains.md`. and also in our [online documentation](https://developers.clever-cloud.com/doc/administrate/log-management/#elasticsearch).

File: bin/clever.js [hunk 2]
```diff
@@ -377,6 +377,12 @@ function run () {
       metavar: 'api_key',
       description: 'Drain custom key',
     }),
+    drainIndexPrefix: cliparse.option('index-prefix', {
+      aliases: ['i'],
+      metavar: 'index_prefix',
+      description: 'Optional drain index prefix for ElasticSearch: `<indexPrefix>-<YYYY-MM-DD>`',
+      default: '`logstash-<YYYY-MM-DD>`',
```
Comment (miton18) [inline bin/clever.js:384]:
```suggestion
      default: 'logstash-<YYYY-MM-DD>`',
``

**Comment by miton18** [comment]
QA OK

File: src/commands/drain.js (hunk 1 above)
Comment (KannarFr) [reply src/commands/drain.js:33]: What do you mean?

**Review by aurrelhebert — APPROVED** [review]

**Pull request merged**